### DISCUSSION TITLE
Eclipse Theia v1.54.0

### DISCUSSION THREAD

**Discussion by sgraband** [eclipse-theia/theia#14222] [Release Announcements] [answered]
Hi everyone 👋,

The Eclipse Theia v1.54.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by tsmaeder** [comment]
Hi @sgraband I'll have to update the API compatilibility to 1.93.1 before the release.

**Reply by tsmaeder** [reply]
https://github.com/eclipse-theia/theia/pull/14224

**Comment by sgraband** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by sgraband** [comment]
The release has completed, thank you to everyone that participated and contributed!

**Reply by tsmaeder** [reply]
Thanks for doing the honors, @sgraband

**Comment by JonasHelming** [comment, marked as answer]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/10/09/eclipse-theia-1-54-release-news-and-noteworthy/)
